Package.swift

```diff
@@ -0,0 +1,88 @@
+// swift-tools-version: 5.6
+//   The swift-tools-version declares the minimum version of Swift required to build this package and MUST be the first
+//   line of this file. 5.6 is required to support zip files for the pod archive binaryTarget.
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+//
+// A user of the Swift Package Manager (SPM) package will consume this file directly from the ORT github repository.
+// For context, the end user's config will look something like:
+//
+//     dependencies: [
+//       .package(url: "https://github.com/microsoft/onnxruntime", branch: "rel-1.15.0"),
```

Comment:
We should update to use a version (instead of `branch:`) and create a release with tag "1.15.0" to validate it works. SPM supposedly works with semver formatted version strings so the usage should change to something like

```swift
    dependencies: [
        .package(url: "https://github.com/microsoft/onnxruntime-swift-package-manager.git", from: "1.15.0"),
    ],
```

Reply:
yes, I think after this change get checked-in, then we can create a release with tag "1.15.0", and include the repo url with version string vs branch name.